cmd/collectors/zapi/plugins/qtree/qtree.go

```diff
@@ -61,10 +62,28 @@ func (my *Qtree) Init() error {
 	my.data = matrix.New(my.Parent+".Qtree", "quota", "quota")
 	my.instanceKeys = make(map[string]string)
 	my.instanceLabels = make(map[string]*dict.Dict)
+	my.historicalLabels = false
 
-	objects := my.Params.GetChildS("objects")
-	if objects == nil {
-		return errs.New(errs.ErrMissingParam, "objects")
+	if my.Params.HasChildS("historicalLabels") {
+		exportOptions := node.NewS("export_options")
+		instanceKeys := exportOptions.NewChildS("instance_keys", "")
+
+		// apply all instance keys, instance labels from parent (qtree.yaml) to all quota metrics
+		//parent instancekeys would be added in plugin metrics
+		for _, parentKeys := range my.ParentParams.GetChildS("export_options").GetChildS("instance_keys").GetAllChildContentS() {
```

Comment:
are we sure that each of these nodes will be present, otherwise we will panic on nil

Reply:
It's export section from qtree template, it should always be there to define which keys/labels would be exported for qtree.
 
<img width="227" alt="image" src="https://user-images.githubusercontent.com/83282894/216626483-d6bf85e1-efae-4619-9f01-c19dec20ccf7.png">

Reply:
added